main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -85,13 +83,11 @@ assert(bobIssuers.Asset === moolaIssuer, details`unexpected Asset issuer`);
 assert(bobIssuers.Price === simoleanIssuer, details`unexpected Price issuer`);
 assert(moolaAmountMath.isEqual(bobInvitationValue.asset, moola(3)), details`wrong asset`);
 assert(simoleanAmountMath.isEqual(bobInvitationValue.price, simoleans(7)), details`wrong price`);
-
 ```
 
-Bob decides to exercise the invitation. He escrows his payments and uses
-his invite to make an offer in the same way as Alice, but his `Proposal` is
-designed to match Alice's (notice that the `give` and `want` clauses are
-reversed from Alice's proposal):
+Bob exercises the invitation. He escrows his payments and uses
+his invitation to make an offer the same way as Alice, but his `Proposal` matches
+Alice's (note that the `give` and `want` clauses are reversed from Alice's proposal):
```

Comment:
I don't like the simple "his proposal matches Alice's", as it doesn't emphasize that is has the terms reversed. What do you of "his proposal is the converse of Alice's"?  Too mathy? I don't like "opposite".  I could go with "his proposal counters Alice's", but it sounds more antagonistic than cooperative. I'll leave it to your judgement, but I like "designed to match".